scene/animation/tween.cpp

```diff
@@ -451,6 +460,7 @@ void Tween::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("bind_node", "node"), &Tween::bind_node);
 	ClassDB::bind_method(D_METHOD("set_process_mode", "mode"), &Tween::set_process_mode);
 	ClassDB::bind_method(D_METHOD("set_pause_mode", "mode"), &Tween::set_pause_mode);
+	ClassDB::bind_method(D_METHOD("set_ignore_time_scale", "ignore"), &Tween::set_ignore_time_scale, DEFVAL(true));
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("set_ignore_time_scale", "ignore"), &Tween::set_ignore_time_scale);
```
I don't think a default makes sense here, but if we do go with it the method declaration should match so it can be used in both extensions and modules with the same signature of `set_ignore_time_scale()`

Reply:
This is similar to `set_parallel()` that also has the default to `true` to as... I assume as a shorthand. 

What @AThousandShips means is that the signature should be
`Ref<Tween> set_ignore_time_scale(bool p_ignore = true);`, but that would **also** be inconsistent with `set_parallel()` that doesn't do this, for some reason.